### PR TITLE
Vimal fix bug1

### PR DESCRIPTION
**Pull request by vimalapugazhan**
Fix low stock update UG and some minor changes. Add comment for future edits of dates in report function in UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ Here are your found items:
 <br>
 
 ### Update an item: `update`
-Update the quantity and/or price of an item
+Update the quantity ,price and/or expiry of an item
```
Comment (awesomesjh) [inline docs/UserGuide.md:158]:
```suggestion
Update the quantity, price and/or expiry date of an item
```

**Comment by vimalapugazhan** [comment]
#166 #150 Fix

File: docs/UserGuide.md [hunk 2]
```diff
@@ -249,12 +251,14 @@ There are 3 items low on stocks!
    Quantity: 30
 ```
 Example: `report r/expiry`
+
+[//]: # (updates dates to be valid for PE day)
 ```
 There are 2 items close to expiry!
 1. Name: Orange
-   Expiry Date: 08/04/2024
+   Expiry Date: 08/04/2024
```
Comment (awesomesjh) [inline docs/UserGuide.md:259]:
```suggestion
   Expiry Date: 08/04/2024
```

**Review by awesomesjh — CHANGES_REQUESTED** [review]
Please take a look at my suggestions

**Comment by vimalapugazhan** [comment]
Done sir

**Pull request merged**